.github/workflows/docker-image-saas-test.yml

```
@@ -0,0 +1,94 @@
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    environment: Build
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)